src/IO/lbsLoadReaderVT.py

```diff
@@ -132,53 +132,65 @@ def read(self, node_id: int, phase_id: int = -1, comm: bool = False) -> tuple:
         return iter_map, comm
 
     def read_iteration(self, n_p: int, phase_id: int) -> list:
-        """Read all the data in the range of procs [0..n_p) for a given
+        """Read all the data in the range of ranks [0..n_p) for a given
         iteration `phase_id`. Collapse the iter_map dictionary from `read(..)`
         into a list of ranks to be returned for the given iteration.
         """
 
         # Create storage for ranks
-        procs = [None] * n_p
-        communication = dict()
+        rank_list = [None] * n_p
+        communications = {}
```

Comment:
`{}` could define set as well. Please leave a `dict()`, that is explicit.

Reply:
Nope, when you use empty curly braces, it is not ambiguous, it is always an empty dict (and **not** an empty set). For an empty set you **must** use `set()`:
https://docs.python.org/2/tutorial/datastructures.html#sets

Furthermore, using `dict()` instead of `{}` results in much slower performance:
http://katrin-affolter.ch/Python/Python_dictionaries

Reply:
ok, may fault, sorry, when it's empty, it will be a `dict`, but I would still insist regarding readability